marshmallow_jsonapi/fields.py

```diff
@@ -71,6 +71,8 @@ def __init__(
         self.id_field = id_field or self.id_field
         super(Relationship, self).__init__(**kwargs)
         self.dump_only = kwargs.pop('dump_only', True)
+        if many and not self.dump_only:
+            raise ValueError('Loading is not supported for many=True.')
```

Comment:
Yes, I think there will be such use cases.
At the time I didn't know of a convenient way to instantiate a model with a collection of ids for a to-many relationship in sqlalchemy. Having seen that recently, I would like to see loading many ids supported and can add that.